packages/radio/src/react/button.js

```diff
@@ -1,24 +1,24 @@
-import { compose, css } from 'glamor'
+import { compose, css, checked } from 'glamor'
 import PropTypes from 'prop-types'
-import React from 'react'
+import React, { useState } from 'react'
 
 import filterReactProps from '@pluralsight/ps-design-system-filter-react-props'
 import Halo from '@pluralsight/ps-design-system-halo'
 import { useTheme } from '@pluralsight/ps-design-system-theme'
-
+import { combineFns } from '@pluralsight/ps-design-system-util'
+import { useRadioContext } from './context.js'
 import stylesheet from '../css/index.js'
 
 const styles = {
   button: () => css(stylesheet['.psds-radio-button']),
-  circle: (themeName, { checked }) =>
+  circle: (themeName, checked) =>
```

Comment:
we've been trying to standardize on the params in these style calls. it usually looks like this

`(themeName, objectContainingStateOrProps) => `

please update all functions to match the pattern

Reply:
That isn't taking props. I mean i can p[ass it a object and then let is access the checked key/value from it but that seems inefficient.  Also every time we do that it's inefficient if we've already desctrutured the value out form props assuming that is where it is coming from.

Reply:
it is taking state which is the patterns we've been using in other places. i could also make the argument around refactor/extensibility around passing objects instead of literal fixed types that lead to prop creep. i'm actually ok if you want to disregard my suggestions and merge as is. eventually we will rethink css so all these patterns will change